faq/app.shapeshift/blockchains.md

```diff
@@ -0,0 +1,77 @@
+
+{% embed url="https://litecoin.org/" %}
+
+### Bitcoin Cash
+
+Bitcoin Cash brings sound money to the world, fulfilling the original promise of Bitcoin as "Peer-to-Peer Electronic Cash". Merchants and users are empowered with low fees and reliable confirmations. The future shines brightly with unrestricted growth, global adoption, permissionless innovation, and decentralized development.
+
+{% embed url="https://bitcoincash.org/" %}
+
+### COSMOS
+
+Cosmos is an ever-expanding ecosystem of interconnected apps and services, built for a decentralized future. Cosmos apps and services connect using IBC, the Inter-Blockchain Communication protocol. This innovation enables you to freely exchange assets and data across sovereign, decentralized blockchains.
+
+{% embed url="https://cosmos.network/" %}
+
+### Avalanche
+
+Avalanche is an open, programmable smart contracts platform for decentralized applications. Build fast, low cost, Solidity-compatible dApps. Launch customized blockchains, private & public. Scale to millions of validators with minimal hardware.
+
+\*Avalanche is supported on ShapeShift only when connected to MetaMask currently. Support for Native and other wallets coming soon.&#x20;
+
+{% embed url="https://www.avax.network/" %}
+
+### Thorchain
+
+[THORChain](https://twitter.com/thorchain) is a multi-chain decentralised exchange ([DEX)](https://www.xdefi.io/article/dex-decentralised-exchanges/) that aims to solve one of the largest issues plaguing the blockchain space: fragmented liquidity spread across different chains. THORChain is a [Cosmos-based protocol](https://v1.cosmos.network/sdk) that allows users to exchange and earn a yield on Layer-1 (native) cryptocurrencies across different blockchains ([BTC](https://coinmarketcap.com/currencies/bitcoin/), [ETH](https://coinmarketcap.com/currencies/ethereum/), [LTC](https://coinmarketcap.com/currencies/litecoin/), [DOGE](https://coinmarketcap.com/currencies/dogecoin/), [BCH](https://coinmarketcap.com/currencies/bitcoin-cash/), [BNB](https://coinmarketcap.com/currencies/bnb/) currently) without going through a centralized exchange. Since the protocol is chain-agnostic, any new native asset can be added to the DEX in future upgrades if there is support for its addition to THORChain.
+
+{% embed url="https://thorchain.com/" %}
+
+### BNB
+
+“BNB stands for “Build the community, and let the community build, build, ’n build.”
+
+The recently released BNB Chain comprises the BNB Beacon Chain for BNB Chain governance (i.e., staking and voting) and the BNB Smart Chain, which remains [Ethereum Virtual Machine (EVM) compatible](https://cointelegraph.com/ethereum-for-beginners/architectural-components-of-the-ethereum-blockchain-what-are-they).
+
+The BNB Chain aims to increase interoperability and develop the underlying infrastructure of the “world’s parallel virtual environment,” a step forward in Binance’s recent MetaFi efforts.
+
+{% embed url="https://www.bnbchain.org/en" %}
+
+### OSMOSIS
```

Comment:
Why caps?

Reply:
I wasn't paying attention. In vscode the folder is correct it won't recognize the change and I did a push just to verify. What's the easiest way to refresh this my git console commands are rusty

Reply:
If you rename the folder and then in a terminal check that the changes are registered with `git status`.

Then add them either explicitly by file name with `git add <path>`, or add all changes `git add *`.

I can't push to your repo, but:

<img width="646" alt="Screenshot 2022-11-11 at 2 06 13 pm" src="https://user-images.githubusercontent.com/97164662/201254190-1fae7588-37bb-45ac-ab75-2b3493479ecd.png">

Reply:
You'll then need to resolve merge conflicts.